eth/filters/api.go

```diff
@@ -218,15 +219,25 @@ func (s *PublicFilterAPI) newLogFilter(earliest, latest int64, addresses []commo
 	filter.SetEndBlock(latest)
 	filter.SetAddresses(addresses)
 	filter.SetTopics(topics)
-	filter.LogCallback = func(log *vm.Log, removed bool) {
+
+	emitLog := func(log *vm.Log, removed bool) {
```

Comment:
Of course if you can rationalise the decision behind separating them I'm happy to hear it :-D